### ISSUE TITLE
Incompatability with macOS Big Sur

### ISSUE THREAD

**Issue by OskarTegby** [dealii/candi#171]
Currently, candi will not compile automatically unless you manually specify your operating system to be macOS Catalina by linking to the folder of supported operating systems. This is an easy fix of course, but nevertheless an issue that could be resolved.

**Comment by koecher** [comment]
Dear @OskarTegby that is correct. We had a lot of trouble with an updated catalina and afterwards we didn't got candi to run for some packages fine. This is why we currently doesn't support MacOS and especially BigSur.

Can you give a list of packages you've installed without problems? Do you have other installation notes (compared to the readme?

**Comment by OskarTegby** [comment]
Dear @koecher. I see. After seeing your message, I aborted the installation and installed it using the dmg file on the installation page https://www.dealii.org/download.html instead, which proved successful. (Albeit with the requirement of downloading Xcode from the Mac App Store despite having the command-line tools installed already.)

However, when I attempted to install it earlier, it recall that just a few packages didn't seem to work (when trying to run it for a few examples in the tutorial). As I now have installed dealii in another way, the issue on is resolved for me. I could, of course, try to install it using candi to provide you with the requested information. Would that remain helpful to you?
